### DISCUSSION TITLE
Intellij

### DISCUSSION THREAD

**Discussion by TonyStarkShaDiao** [community/community#7179] [Copilot Conversations]
可以麻烦尽快把候选人资格加上吗?没有资格根本体验不了

**Comment by ICitiesz** [comment]
您好，你的体验权限即将来临，请您耐心等待。获得体验权限的时间大概需要等4+个月，并且要看你是否为其他人作品做出多少的贡献来获取更早的体验权限。

Hello there, the preview will coming soon to you, please be patience. Getting the preview may take approximately 4+ months, it also depends on how much you contribute on others people project to get the earliest preview.
